Source/WebKit/GPUProcess/media/cocoa/RemoteMediaPlayerProxyCocoa.mm

```diff
@@ -68,12 +98,20 @@
     if (layer && !m_inlineLayerHostingContext) {
         m_inlineLayerHostingContext = LayerHostingContext::createForExternalHostingProcess();
         IntSize presentationSize = enclosingIntRect(FloatRect(layer.frame)).size();
+#if USE(EXTENSIONKIT)
+        sendRenderingModeChangedXPCMessage(presentationSize);
```

Comment:
So we now message the UIProcess, which in turn messages the WebProcess, rather than the old mechanism of direct message sends to the WCP.

Is there no way for us to directly communicate between Extensions? I guess maybe only the parent process knows all of the children and can be the exchange for relaying information.

Reply:
Per Arne and I chatted about this offline. I think we actually do want to send the layer hosting context over CoreIPC for IPC validation purposes. That would have the nice side effect of letting us send the message directly from GPU to WC without having to create a direct XPC connection. @beidson can speak more to how/why we should use CoreIPC here.